fhir-server/src/test/java/au/csiro/pathling/fhirpath/parser/ParserTest.java

```diff
@@ -177,23 +177,20 @@ public void testInOperator() {
 
   @Test
   public void testCodingOperations() {
+    // Check that membership operators for codings use strict equality rather than equivalence.
     // test unversioned
     assertThatResultOf(
         "maritalStatus.coding contains http://terminology.hl7.org/CodeSystem/v3-MaritalStatus|S")
         .isElementPath(BooleanPath.class)
         .selectOrderedResult()
-        .hasRows(allPatientsWithValue(spark, true)
-            .changeValue(PATIENT_ID_8ee183e2, false)
-            .changeValue(PATIENT_ID_9360820c, false)
-            .changeValue(PATIENT_ID_beff242e, false));
+        .hasRows(allPatientsWithValue(spark, false));
 
     // test versioned
     assertThatResultOf(
         "http://terminology.hl7.org/CodeSystem/v2-0203|v2.0.3|PPN in identifier.type.coding")
```

Comment:
Is the version in the wrong position here?

Reply:
Yes. It's.